projects/ngxpert/hot-toast/src/lib/components/hot-toast/hot-toast.component.ts

```diff
@@ -160,7 +161,7 @@ export class HotToastComponent implements OnInit, AfterViewInit, OnDestroy, OnCh
     this.groupRefs = value;
 
     // maybe below will prevent execution in ngDoCheck?
-    (this.toastRef as { groupRefs: CreateHotToastRef<unknown>[] }).groupRefs = value;
+    (this.toastRef() as { groupRefs: CreateHotToastRef<unknown>[] }).groupRefs = value;
```

Comment:
This is not the right way to update signal.

Reply:
Input signals are read-only inside the component that receives them. We can use the model input feature available in 17.2 version

Reply:
Let me check this.